dockstore-webservice/src/main/java/io/dockstore/webservice/core/ChecksumConverter.java

```diff
@@ -0,0 +1,50 @@
+package io.dockstore.webservice.core;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import javax.persistence.AttributeConverter;
+import javax.persistence.Converter;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+@Converter(autoApply = true)
+public class ChecksumConverter implements AttributeConverter<List<Checksum>, String> {
+
+    private static final Logger LOG = LoggerFactory.getLogger(ChecksumConverter.class);
+    @Override
+    public String convertToDatabaseColumn(List<Checksum> checksums) {
+        String stringChecksums;
+        if (checksums != null && !checksums.isEmpty()) {
+            try {
+                stringChecksums = checksums.stream().map(checksum -> checksum.getType().trim() + ":" + checksum.getChecksum().trim()).collect(Collectors.joining(","));
```

Comment:
This is personal preference, but I would just return the value here directly. Then you can get rid of the variable declaration on line 19, as well as line 27. And maybe line 25, but see my comment there.